src/utils/axios-instance.ts

```diff
@@ -17,4 +17,4 @@ instance.interceptors.response.use(
   error => Promise.reject(error)
 )
 
-export { instance as axios }
+export { instance as axiosInstance }
```

Comment:
Renamed (and also renamed this file) to differentiate this from the unmodified axios library import.